### PR TITLE
first repo

### PR DESCRIPTION
**Pull request by Julia-69**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"
+            integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49"
+            crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/js/bootstrap.min.js"
+            integrity="sha384-o+RDsa0aLu++PJvFqy8fFScvbHFLtbvScb8AjopnFD+iEQ7wo/CG0xlczd+2O/em"
+            crossorigin="anonymous"></script>
+
+</head>
+<body>
+/*  HEADER */
```
Comment (NazarShershen) [inline index.html:30]: Цей текст виводиться на сторінці. Це php коментар, коментарі в html записуються по-іншому

File: index.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    <div class="col-lg-7 col-md-3 col-sm-2">
+       <img src="img/logo.png" alt="Logo" class="logo">
+    </div>
+    <div class="col-lg-5 col-md-5 col-sm-2">
+        <nav class="navbar navbar-expand-lg mr-auto">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div class="navbar-nav">
```
Comment (NazarShershen) [inline index.html:43]: чому пункти меню не лінки? не клікабельні?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    </div>
+  </div>
+    </div>
+    <div class="container">
+    <div class="row">
+        <div class="col-lg-8 col-md-2 col-sm-2 offset-md-2">
+        <h1><span class="font-weight-bold text-center">Quick Search for Your<strong class="color"> Perfect Product</strong></span></h1>
+        <p class="text-uppercase offset-md-4" id="text-center"><small>Innovate &#8226; List &#8226; Connect</small></p>
+                <div class="input-group">
+                    <div class="input-group-text bg-light" id="button1">
```
Comment (NazarShershen) [inline index.html:61]: погана назва ідентифікатора, з назви нічого не ясно.

Також негарний ефект при ховері, змінюєтсья колір кнопки і не змінюється колір в батьківському блоціБ і виглядає не гарно

File: index.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                            <a class="dropdown-item" href="#">Telegram</a>
+                            <div role="separator" class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="#">Skype</a>
+                            <div role="separator" class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="#">Viber</a>
+                        </div>
+                    </div>
+                        <input type="text" class="form-control" aria-label="Text input with dropdown button">
+                    <div class="input-group-text bg-info" id="button2">
+                        <button class="btn btn-sm btn-default text-uppercase text-light bg-info" type="button"><small>
```
Comment (NazarShershen) [inline index.html:76]: те ж саме з ховером.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+                            <strong>Start searching<strong></small></button>
```
Comment (NazarShershen) [inline index.html:77]: для чого пхати текст в тег small, який задає звичайну жирність і всередині нього робити тег strong, який робить текст жирним?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+                    </div>
+                </div>
+            <div class="input-group-prepend offset-md-2">
```
Comment (NazarShershen) [inline index.html:80]: кнопки плавають, на різних моніторах відображаються в різних місцях

File: index.html [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+            </div>
+        </div>
+    </div>
+</div>
+</header>
+
+<div class="container">
+    <div class="row">
+        <div class="col-lg-8 col-md-2 col-sm-2 offset-md-2">
+            <h3 class="text-center"> Most Popular Products </h3>
```
Comment (NazarShershen) [inline index.html:92]: нащо такий величезний padding?

Через те, що кнопки All Categories i How it works розмішені погано, цей елемент також зроблений як небудь

File: index.html [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+            <p class="text-center text-muted"> This is the list of products that are most interested in users </p>
+        </div>
+    </div>
+</div>
+
+<div class="row justify-content-center">
+    <div class="col-md-8">
+        <div class="row">
+            <a href="img" data-toggle="lightbox" data-gallery="example-gallery" class="col-sm-4">
```
Comment (NazarShershen) [inline index.html:101]: бачу класи з lightbox, але картинки не відкриваються для перегляду

File: index.html [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+                <img src="img/image2.jpg" class="img-fluid">
+                <small class="text-muted">Jira 2017</small>
+            </a>
+            <a href="img" data-toggle="lightbox" data-gallery="example-gallery" class="col-sm-4">
+                <img src="img/image3.jpg" class="img-fluid">
+                <small class="text-muted">Slack Pro</small>
+            </a>
+        </div>
+        <br>
```
Comment (NazarShershen) [inline index.html:114]: такі відступи треба задавати стилями а не <br>

File: index.html [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            <nav class="navbar navbar-expand-lg navbar-light">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler"></span>
+                </button>
+                <div class="navbar-collapse collapse justify-content-between align-items-center w-100">
+                    <div class="navbar-nav text-center" id="collapsingNavbar2">
+                        <p class="h6 mb-3 nav-item nav-link text-uppercase text-light"><strong>About us &nbsp; &nbsp; &nbsp;</strong></p>
```
Comment (NazarShershen) [inline index.html:154]: так само як в хедері, це мають бути клікабельні лінки

File: index.html [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+<div class="row">
+    <div class="col-lg-6 col-md-4 col-sm-2 offset-md-3">
+        <p class="text-center" id="link">All products</p>
+    </div>
+</div>
+
+<div class="container-fluid">
+    <div class="row">
+                <img class="mx-auto d-block" src="img/icon_bottom.png" alt="Icon" id="icon">
+        <div class="col-lg-12 col-md-12 col-sm-12 offset-md-3">
```
Comment (NazarShershen) [inline index.html:142]: поганий спосіб центрування. Меню не по центру. Є flexbox і для центрування його треба використовувати

File: style.css [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+
+span .color{
+    color: green;
+}
+
+.span2{
+    width: 30vw;
+}
+
+.{
```
Comment (NazarShershen) [inline style.css:80]: яка функція цього запису?

File: index.html [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+        <nav class="navbar navbar-expand-lg mr-auto">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div class="navbar-nav">
+        <p class="nav-item nav-link text-uppercase"><small> &nbsp; &nbsp; All products &nbsp; &nbsp;</small></p>
+        <p class="nav-item nav-link text-uppercase"><small>About us &nbsp; &nbsp;</small></p>
+        <p class="nav-item nav-link text-uppercase"><small>Log in &nbsp; &nbsp;</small></p>
+        <p class="nav-link "><span class="line"></span></p>
```
Comment (NazarShershen) [inline index.html:47]: це має бути не окремим пунктом меню, а зробити треба через псевдоклас :before або :after

File: index.html [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+       <img src="img/logo.png" alt="Logo" class="logo">
+    </div>
+    <div class="col-lg-5 col-md-5 col-sm-2">
+        <nav class="navbar navbar-expand-lg mr-auto">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div class="navbar-nav">
+        <p class="nav-item nav-link text-uppercase"><small> &nbsp; &nbsp; All products &nbsp; &nbsp;</small></p>
```
Comment (NazarShershen) [inline index.html:44]: Нащо там стільки тих пробілів?

File: index.html [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+                        </div>
+                    </div>
+                        <input type="text" class="form-control" aria-label="Text input with dropdown button">
+                    <div class="input-group-text bg-info" id="button2">
+                        <button class="btn btn-sm btn-default text-uppercase text-light bg-info" type="button"><small>
+                            <strong>Start searching<strong></small></button>
+                    </div>
+                </div>
+            <div class="input-group-prepend offset-md-2">
+                <span class="input-group-text bg-light " id="btn"><span class="add-on-one btn-md"><i class="fas fa-th"></i></span> &nbsp; All categories</span>
```
Comment (NazarShershen) [inline index.html:81]: не може бути два однаковий id на сторінці. На то він і id, щоб бути унікальним

File: index.html [hunk 16]
```diff
@@ -0,0 +1,182 @@
...
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-4 col-sm-2 offset-md-5">
+                <br><p class="text-white md-auto d-block"><small> &copy; 2018 Find Parrot</small></p>
+            </div>
+        </div>
+    </div>
+</footer>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (NazarShershen) [inline index.html:177]: нащо два рази підключати jquery i bootstrap? Підключення вже було в хедері

File: style.css [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+.nav-link{
+    color: #434343 !important;
+    font-size: 13px;
+}
+
+.navbar-collapse{
+    font-size: 10px;
+}
+#text-center{
+    color: #434343;
```
Comment (NazarShershen) [inline style.css:62]: колір повторюється. Не краще зробити окремий клас на це колір і присвоювати його елементам ніж дублювати стиль?

File: style.css [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+#button1 {
+    border-radius: 10px 0 0 10px;
+}
+
+#button2 {
+    border-radius: 0 10px 10px 0;
+}
+
+#btn{
+    padding-top: 25px;
```
Comment (NazarShershen) [inline style.css:136]: чому не задати всі педінги одним стилем padding: 25px 30px;

File: style.css [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+    color: forestgreen;
+    margin-left: 31px;
+}
+
+.input-group-prepend{
+    padding-top: 30vh;
+}
+
+.fa-th{
+    color: forestgreen;
```
Comment (NazarShershen) [inline style.css:151]: дублювання

**Review by NazarShershen — CHANGES_REQUESTED** [review]
Залишив коментарі в коді. Також адаптивності немає. На малих екранах все пливе, немає навігації для малих екранів. Переглянь як виглядає сторінка в емуляторі девайсів, якщо не знаєш що це і як користуватися, запитай.
